src/chatview/styles.js

```diff
@@ -0,0 +1,54 @@
+const styles = (theme) => ({
+  content: {
+    height: "calc(100vh - 100px)",
+    overflow: "auto",
+    padding: "25px",
+    marginLeft: "300px",
+    boxSizing: "border-box",
+    overflowY: "scroll",
+    top: "50px",
+    width: "calc(100% - 300px)",
+    position: "absolute",
+  },
+
+  userSent: {
+    float: "left",
```

Comment:
I'm seeing a few `float`s and `clear`s in your styles. Unfortunately I'm unable to view this in the browser without the data so I can't be sure what these styles look like. Have you tried using CSS flexbox yet? It's a much more modern way of styling layouts. Since using it I've never had to go back to using floats and clears.